### PR TITLE
[GNA] Enabled new OV inference tests

### PR DESCRIPTION
**Pull request by mryzhov**
### Details:
 - Enabled new InferRequest behavior tests with OV 2.0
 - Tests with dynamic cases are skipped because GNA plugin doesn't support them
 - Tests with cancelling of infer requests are skipped because it was not implemented yet (71068)
 - 1 chaining test failed due to bug(71070), marked as skipped.

### Tickets:
 - 69194

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

File: inference-engine/tests/functional/plugin/gna/shared_tests_instances/behavior/ov_infer_request/io_tensor.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    ov::element::i8,
+    ov::element::i16,
+    ov::element::i32,
+    ov::element::i64,
+    ov::element::u1,
+    ov::element::u4,
+    ov::element::u8,
+    ov::element::u16,
+    ov::element::u32,
+    ov::element::u64,
```
Comment (nkogteva) [inline inference-engine/tests/functional/plugin/gna/shared_tests_instances/behavior/ov_infer_request/io_tensor.cpp:33]: Currently, we support only f32, i16, u8 precisions. So I'm wondering how it should work.

File: inference-engine/tests/functional/plugin/gna/shared_tests_instances/skip_tests_config.cpp [hunk 2]
```diff
@@ -78,5 +83,9 @@ std::vector<std::string> disabledTestPatterns() {
         // TODO: Issue:27391
         // TODO: Issue:28036
         R"(.*ActivationLayerGNATest.*(Log|Exp).*netPRC=(FP16|FP32).*)",
+        // TODO: Issue: CVS-71068
```
Comment (nkogteva) [inline inference-engine/tests/functional/plugin/gna/shared_tests_instances/skip_tests_config.cpp:86]:
```suggestion
        // TODO: Issue: 71068
```

File: inference-engine/tests/functional/plugin/gna/shared_tests_instances/skip_tests_config.cpp [hunk 3]
```diff
@@ -78,5 +83,9 @@ std::vector<std::string> disabledTestPatterns() {
...
+        R"(.*OVInferRequestCancellationTests.*)",
+        // TODO: Issue: CVS-71070
```
Comment (nkogteva) [inline inference-engine/tests/functional/plugin/gna/shared_tests_instances/skip_tests_config.cpp:88]:
```suggestion
        // TODO: Issue: 71070
```

Comment (mryzhov) [reply inference-engine/tests/functional/plugin/gna/shared_tests_instances/skip_tests_config.cpp:88]: oh missed that, thanks

File: inference-engine/tests/functional/plugin/gna/shared_tests_instances/behavior/ov_infer_request/io_tensor.cpp (hunk 1 above)
Comment (mryzhov) [reply inference-engine/tests/functional/plugin/gna/shared_tests_instances/behavior/ov_infer_request/io_tensor.cpp:33]: shared test verify that exception will be thrown for unsupported precisions

**Review by nkogteva — APPROVED** [review]

**Pull request merged**